docs/source/models/gbd2017/causes/ischemic_heart_disease/index.rst

```diff
@@ -11,37 +11,65 @@ Ischemic heart disease (IHD) is a non-communicable cardiovascular disease which
 
 Acute coronary events may cause symptoms such as angina, cold sweats, dizziness, nausuea, neckpain, shortness of breath, sleep disturbances, or weakness. 
```

Comment:
Above you mention that IHD is a leading cause of YLLs. Is it also a significant cause of YLDs?